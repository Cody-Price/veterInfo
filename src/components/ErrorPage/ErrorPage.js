import React from 'react'
import './ErrorPage.scss'

const ErrorPage = () => {
  return (
    <div className="error-page">
      Error 404: the page you requested does not exist. <br />
      Please check the address you requested and try again.
    </div>
  )
}

export default ErrorPage