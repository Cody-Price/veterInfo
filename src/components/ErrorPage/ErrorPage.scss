.error-page {
  position: relative;
  top: 300px;
  left: 8vw;
}