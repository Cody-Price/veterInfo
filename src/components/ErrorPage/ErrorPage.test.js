import ErrorPage from './ErrorPage'
import React from 'react'
import { shallow } from 'enzyme'
import { configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })

describe('ErrorPage', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<ErrorPage />)
    expect(wrapper).toMatchSnapshot()
  })
})