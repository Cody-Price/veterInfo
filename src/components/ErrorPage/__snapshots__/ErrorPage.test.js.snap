// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPage />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        "Error 404: the page you requested does not exist. ",
        <br />,
        "Please check the address you requested and try again.",
      ],
      "className": "error-page",
    },
    "ref": null,
    "rendered": Array [
      "Error 404: the page you requested does not exist. ",
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": "br",
      },
      "Please check the address you requested and try again.",
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          "Error 404: the page you requested does not exist. ",
          <br />,
          "Please check the address you requested and try again.",
        ],
        "className": "error-page",
      },
      "ref": null,
      "rendered": Array [
        "Error 404: the page you requested does not exist. ",
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": "br",
        },
        "Please check the address you requested and try again.",
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
